app/assets/stylesheets/components/_button.scss

```diff
@@ -147,7 +147,7 @@
   }
 }
 
-.favorite {
+.favourite {
```

Comment:
Favourites is red then turns to green on the developer page, but is green and stays green for a company page. Could these be made consistent?